sdk/identity/azure-identity/README.md

```diff
@@ -137,6 +137,15 @@ Configuration is attempted in the above order. For example, if values for a
 client secret and certificate are both present, the client secret will be used.
 
 # Examples
+
+Following examples are provided below:
+
+* [Authenticating with DefaultAzureCredential](#authenticating-with-defaultazurecredential)
+* [Authenticating a service principal with a client secret](#authenticating-a-service-principal-with-a-client-secret)
+* [Authenticating a service principal with a certificate](#authenticating-a-service-principal-with-a-certificate)
+* [Chaining credentials](#chaining-credentials)
+* [Async credentials](#async-credentials)
```

Comment:
`sphinx` doesn't parse this syntax as GitHub does, producing broken links on PyPI and our docs site.
```suggestion
The following examples are provided below:

* [Authenticating with DefaultAzureCredential](#authenticating-with-defaultazurecredential "Authenticating with DefaultAzureCredential")
* [Authenticating a service principal with a client secret](#authenticating-a-service-principal-with-a-client-secret "Authenticating a service principal with a client secret")
* [Authenticating a service principal with a certificate](#authenticating-a-service-principal-with-a-certificate "Authenticating a service principal with a certificate")
* [Chaining credentials](#chaining-credentials "Chaining credentials")
* [Async credentials](#async-credentials "Async credentials")
```